src/cli/test/create.js

```diff
@@ -27,8 +29,25 @@ const main = async function(args) {
     }
   }
 
+  if (args.headers) {
+    if (['{', '['].includes(args.headers.substr(0, 1))) {
```

Comment:
Maybe we could use `JSON.parse` in a try / catch block here instead?

Reply:
👍️ Will update